internal/cli/atlas/deployments/list.go

```diff
@@ -47,6 +48,7 @@ const listTemplate = `NAME	TYPE	MDB VER	STATE
 {{end}}`
 
 const MaxItemsPerPage = 500
+const errAtlas = "failed to retrieve Atlas deployments with: %v"
```

Comment:
```suggestion
const errAtlas = "failed to retrieve Atlas deployments with: %s"
```

as you're using atlasErr.Error() which is string